From da03b77a25b0d3b8234152094c9158b067ec5ecb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 27 Jan 2018 00:23:56 -0800
Subject: [PATCH] mse: Print MediaTime consistently using %s

clang finds a mismatched typecast from MediaTime to float to be unacceptable
its fixed already in webkit upstream

Partial backport from https://github.com/WebKit/webkit/commit/6cf9b9ded3de0240fd8efcac812d722b30e3d74b#diff-52262bbefc8558f6ad277f98a4f6ebbe

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 .../platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp b/Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp
index 2753ddb0c63..6e9f4f8e1b2 100644
--- a/Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp
+++ b/Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp
@@ -207,7 +207,7 @@ void MediaPlayerPrivateGStreamerMSE::seek(const MediaTime& time)
     }
 
     m_isEndReached = false;
-    GST_DEBUG("m_seeking=%s, m_seekTime=%f", boolForPrinting(m_seeking), m_seekTime);
+    GST_DEBUG("m_seeking=%s, m_seekTime=%s", boolForPrinting(m_seeking), toString(m_seekTime).utf8().data());
 }
 
 void MediaPlayerPrivateGStreamerMSE::configurePlaySink()
-- 
2.16.1

